### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by cpanderson** [ledermann/unread#125]
Lots of old open issues and mark_as_read! seems to be currently broke. Should we move on?

**Comment by ledermann** [comment]
No, the project isn't dead - but because I'm busy with other projects, I'm not currently working on it. But hey, it's open-source, you are welcome to submit PR's to fix the issues.

I have been using this gem in some of my applications for a very long time, and it still works very well. I' don't see that `mark_as_read!` is broken.

**Comment by cpanderson** [comment]
Oh ok, just see lots of old open issues and a very old PR. 

Yah, can only get the model mark_as_read!(:all, for: user) to work. Associated records or a specific record for user does not as per documentation. No commits or errors sql-wise. On latest 5.2.x Rails and Postgres.
